src/libServer/LookupServer.cpp

```diff
@@ -1147,6 +1158,10 @@ Json::Value LookupServer::TxBlockListing(unsigned int page) {
       m_mediator.m_txBlockChain.GetLastBlock().GetHeader().GetBlockNum();
   Json::Value _json;
 
+  if (currBlockNum == INIT_BLOCK_NUMBER) {
```

Comment:
How about move this error checking to line 1159, just in back of the parameter assignment? This can avoid the potential misuse of this variable in the future.